packages/atlas/src/views/studio/VideoWorkspace/VideoWorkspace.tsx

```diff
@@ -34,17 +38,40 @@ export const VideoWorkspace: React.FC = React.memo(() => {
   const { openWarningDialog } = useDisplayDataLostWarning()
 
   const handleVideoWorkspaceSubmit = useHandleVideoWorkspaceSubmit()
+  const handleNFTWorkspaceSubmit = () => null
+
+  const toggleTermsAccept = () => {
+    setNFTTermsAccepted((prevState) => !prevState)
+  }
 
   const handleSubmit = useCallback(
     (data: VideoFormData) => {
       if (isIssuedAsNFT) {
+        if (NFTCurrentStepIdx === -1) {
+          setVideoFormDataForNFT(data)
+        }
+        if (NFTCurrentStepIdx === 2) {
+          handleVideoWorkspaceSubmit(data)
+          return
+        }
         setNFTCurrentStepIdx((step) => (step === null ? 0 : step + 1))
-        setVideoFormDataForNFT(data)
       } else {
         handleVideoWorkspaceSubmit(data)
       }
     },
-    [handleVideoWorkspaceSubmit, isIssuedAsNFT]
+    [NFTCurrentStepIdx, handleVideoWorkspaceSubmit, isIssuedAsNFT]
+  )
+
+  const handleNFTSubmit = useCallback(
+    // eslint-disable-next-line
```

Comment:
What's the reason for this?

Reply:
I wanted to leave a note that we are using `data` argument here, but since it's not used yet eslint throws warning.

Reply:
Ah I see. You can name the argument `_`, eslint will ignore it then